src/frontend/components/Navbar/NavItem.jsx

```diff
@@ -9,10 +9,13 @@ const NavItem = ({ navItemTitle, number, icon, navigateTo }) => {
       <span className="p-relative">
         <i className={`fas fa-${icon} text-md`}></i>
         {number !== undefined && (
-          <span className="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute">
+          <span
+            className="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute"
+            style={number > 9 ? { padding: "1px 4px" } : { padding: "1px 8px" }}
```

Comment:
Give some generic name to 'number' like item's count. When I see that variable I didn't get immediate idea on what it refers to.

Reply:
done